cloudify_kubernetes/utils.py

```diff
@@ -213,8 +216,22 @@ def resolve_kind():
     return definition
 
 
+def validate_resource_definition(resource_definition):
+    for field in [KIND_DEFINITION, API_VERSION_DEFINITION,
+                  METADATA_DEFINITION]:
+        if field not in resource_definition:
+            raise KuberentesInvalidDefinitionError(
+                'Incorrect format of resource definition,one or more of: {0}, '
+                '{1}, {2} '
+                'are missing.'.format(
+                    API_VERSION_DEFINITION, METADATA_DEFINITION,
+                    KIND_DEFINITION))
+    return resource_definition
+
+
```

Comment:
Why should this be a util and not in __init__ method of https://github.com/cloudify-cosmo/cloudify-kubernetes-plugin/blob/bd51ca4d817c21fcd0681099b72974dfc18f5f51/cloudify_kubernetes/k8s/client.py#L53